mondey_backend/src/mondey_backend/routers/statistics.py

```diff
@@ -195,7 +194,6 @@ def calculate_milestone_statistics_by_age(
         MilestoneAgeScoreCollection object which contains a list of MilestoneAgeScore objects,
     one for each month, or None if there are no answers for the milestoneg and no previous statistics.
     """
-    # TODO: when the answersession eventually has an expired flag, this can go again.
```

Comment:
I'd keep this comment as we still don't have an expired flag for the answer session, but I made a new issue #219 for this